Mage.Sets/src/mage/cards/t/TheAnimus.java

```diff
@@ -0,0 +1,149 @@
+import mage.game.Game;
+import mage.game.permanent.Permanent;
+import mage.players.Player;
+import mage.target.Target;
+import mage.target.Targets;
+import mage.target.common.*;
+import mage.target.targetpointer.FixedTarget;
+import mage.util.functions.EmptyCopyApplier;
+
+/**
+ *
+ * @author grimreap124
+ */
+public final class TheAnimus extends CardImpl {
+
+    private static final FilterCard filter = new FilterCreatureCard("legendary creature card");
+    private static final FilterCard exileFilter = new FilterCreatureCard("creature card in exile with a memory counter on it");
+
+    static {
+        filter.add(SuperType.LEGENDARY.getPredicate());
+        exileFilter.add(CounterType.MEMORY.getPredicate());
+    }
+
+    public TheAnimus(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.ARTIFACT}, "{2}");
+        
+        this.supertype.add(SuperType.LEGENDARY);
+
+        // At the beginning of your end step, exile up to one target legendary creature card from a graveyard with a memory counter on it.
+        Ability ability = new BeginningOfEndStepTriggeredAbility(new TheAnimusEffect(), TargetController.YOU, false);
+        ability.addTarget(new TargetCardInGraveyard(0, 1, filter));
+        this.addAbility(ability);
+
+        // {T}: Until your next turn, target legendary creature you control becomes a copy of target creature card in exile with a memory counter on it. Activate only as a sorcery.
+        ability = new ActivateAsSorceryActivatedAbility(new TheAnimusCopyEffect(), new TapSourceCost());
+        ability.addTarget(new TargetCreaturePermanentSameController(1, StaticFilters.FILTER_CREATURE_LEGENDARY));
+        ability.addTarget(new TargetCardInExile(1, 1, exileFilter));
+        this.addAbility(ability);
+    }
+
+    private TheAnimus(final TheAnimus card) {
+        super(card);
+    }
+
+    @Override
+    public TheAnimus copy() {
+        return new TheAnimus(this);
+    }
+}
+
+class TheAnimusEffect extends OneShotEffect {
+
+    TheAnimusEffect() {
+        super(Outcome.Benefit);
+        staticText = "exile up to one target legendary creature card from a graveyard with a memory counter on it";
+    }
+
+    private TheAnimusEffect(final TheAnimusEffect effect) {
+        super(effect);
+    }
+
+    @Override
+    public TheAnimusEffect copy() {
+        return new TheAnimusEffect(this);
+    }
+
+    @Override
+    public boolean apply(Game game, Ability source) {
+        Player controller = game.getPlayer(source.getControllerId());
+        UUID targetId = getTargetPointer().getFirst(game, source);
+        if (controller == null || targetId == null) {
+            return false;
+        }
+
+        Card card = game.getCard(targetId);
+        if (card == null) {
+            return false;
+        }
+        controller.moveCards(card, Zone.EXILED, source, game);
+        if (game.getState().getZone(card.getId()) != Zone.EXILED) {
+            return false;
```

Comment:
Must return true here. Basic logic in most use cases:
* if effect something changed in game state then return true;
* If nothing changes then return false.